games.json

```diff
@@ -6754,7 +6763,7 @@
     "status": "Broken",
     "reference": "",
     "anticheats": [
-      "NEAC Protect"
+      "NetEase Anti-Cheat Expert"
```

Comment:
Does NACE work in Proton? Thought this game used the other version of the AC which is more compatible with Linux?

Reply:
@Starz0r Yes it works. NACE is just a hybrid between a usermode anti-tamper solution like Hyperion and a server-side anticheat like FairFight.

This game in particular requires Proton 7.0-6 or Proton-GE 7-55 in order to accept the game's EULA/TOS.